### PR TITLE
Best practices for libpacemaker graph consumer functions

### PR DESCRIPTION
**Pull request by kgaillot**
I haven't done a "best practices" PR in awhile (defined narrowly as bringing in line with current project guidelines), so I figured I'd tackle a small corner of libpacemaker.

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , can you review when you get a chance? Low priority

File: lib/pacemaker/pcmk_graph_consumer.c [hunk 1]
```diff
@@ -276,14 +331,17 @@ static crm_graph_functions_t default_fns = {
     pseudo_action_dummy
 };
 
-int
-run_graph(crm_graph_t * graph)
+/*!
+ * \internal
+ * \brief
```
Comment (clumens) [inline lib/pacemaker/pcmk_graph_consumer.c:336]: Missing a description here.

**Comment by clumens** [comment]
Otherwise, this all looks good to me.  I like the more reasonable filenames and how the graph freeing functions look now.

File: lib/pacemaker/pcmk_graph_consumer.c (hunk 1 above)
Comment (kgaillot) [reply lib/pacemaker/pcmk_graph_consumer.c]: lol, updated commit

**Pull request merged**